gwnr/waveform/esigma_utils.py

```diff
@@ -579,10 +579,10 @@ def get_imr_esigma_modes(
         )  # index at which orb_freq becomes just larger than transition orbital
         # frequency, towards the end of waveform
 
-        if verbose:
+        if verbose > 4:
             print(
-                f"""The transition is to happen at index: {transition_idx}, with
-                length: {len(orb_freq)}"""
+                f"""The transition is to happen in a window centered at
+                index: {transition_idx}. Total length is: {len(orb_freq)}"""
```

Comment:
At this point in the code, one can't say whether `transition_idx` will be at the window's centre or its end.  I think a better statement that is also true to what's happening in the code would be 

```print(f"""Transition frequency found at index {transition_idx} in the orbital frequency array of length {len(orb_freq)}")```

Reply:
rephrased as suggested